lib/devise-security/models/expirable.rb

```diff
@@ -22,7 +22,8 @@ module Expirable
 
       # Updates +last_activity_at+, called from a Warden::Manager.after_set_user hook.
       def update_last_activity!
-        self.update_column(:last_activity_at, Time.now.utc)
+        self.last_activity_at = Time.now.utc
```

Comment:
Fix the CC issue here

Reply:
There is a subtle difference here in behavior.  The original implementation  bypasses validation checks, while this approach does not.  It's possible for this call to fail if the record is in an invalid state for some reason.